composer.json

```diff
@@ -21,7 +21,7 @@
         "drupal/devel": "^1.2",
         "drupal/drupal-extension": "^4.0.0@alpha",
         "drush/drush": "^9",
-        "openeuropa/code-review": "^0.2",
+        "openeuropa/code-review": "^0.3",
```

Comment:
This is updating the version string of the code review tool to 0.3 but it is in fact not using the features that are specific to that version, so this still works fine with 0.2. Let's change this instead to `~0.2` so that we clearly communicate that we are compatible with both 0.2, 0.3 and future versions.